ReactiveCocoa/Swift/Property.swift

```diff
@@ -94,7 +94,13 @@ public final class MutableProperty<Value>: MutablePropertyType {
 	/// `value`. Note that recursive sets will still deadlock because the
 	/// underlying producer prevents sending recursive events.
 	private let lock: NSRecursiveLock
-	private var _value: Value
+
+	/// The getter of the underlying storage, which may outlive the property
+	/// if a returned producer is being retained.
+	private let getter: () -> Value
+
+	/// The setter of the underlying storage.
+	private let setter: Value -> Void
```

Comment:
I am following the convention of the Swift corelibs and stdlib of using `Void` instead of `()` for function types.